app/konnect/runtime-manager/composite-runtime-groups/how-to.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Working with composite runtime groups
+content_type: how-to
+badge: enterprise
+---
+
+A composite runtime group (CRG) is a type of runtime group in {{site.konnect_short_name}} where `cluster_type = CLUSTER_TYPE_COMPOSITE`.
```

Comment:
I think maybe this sentence could steal from the composite group overview intro paragraph because I was kind of confused about the `cluster_type` thing (and it didn't explain what a composite runtime group was).